packages/legacy/src/Mappings/components/helpers.ts

```diff
@@ -24,12 +24,17 @@ import { getBuilderItemsFromMapping } from './MappingBuilder/helpers';
 import { ProviderType } from 'legacy/src/common/constants';
 import { getStorageTitle } from 'legacy/src/common/helpers';
 
+const domainAndNameToName = (domainnAndName: string): string => {
+  const match = domainnAndName.match(/\/([-a-z0-9]*)\/?$/);
+  return match ? match[1] : domainnAndName;
```

Comment:
```suggestion
  return domainnAndNam.split('/')?.[1] || domainnAndNam;
```
can you try split ?

https://github.com/kubev2v/forklift/blob/main/pkg/controller/plan/adapter/ocp/validator.go#L155